### ISSUE TITLE
Add filter for branch name

### ISSUE THREAD

**Issue by Wesztman** [expertasolutions/AzDo.VstsDashboard#189]
Would be nice to be able to select certain branches (more than one) similar to the project setting :) 

![image](https://github.com/expertasolutions/AzDo.VstsDashboard/assets/54413402/0a50f396-23a2-4a29-9702-2d61a1f80126)

I want to have this on a big screen in our office, but I only want to show the status och "main" and "master" branches :) And while at it the same for repos would also be nice so that I can select a subset of repos which are interesting :)

I would love to contribute myself, but I'm not very good at typescript :(

Cheers 
Carl

**Comment by lturmel** [comment]
Hi @Wesztman, that is an interesting request, and I will try to find some time, later during march, to look into it.

thank your for your request

**Comment by Wesztman** [comment]
@lturmel 

Awsome! 😄 Thank you for taking the time to develop this plugin 😊 and responding to issues/feature requests, much appreciated 🥰

**Comment by AshishDadhich4h2** [comment]
@lturmel yeah that branch filter option will be very helpful as specially when we want to track/monitor only prod releases.
